source/limitations.txt

```diff
@@ -0,0 +1,96 @@
+.. _entity-framework-limitations:
+
+===========
+Limitations
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta:: 
+   :keywords: .NET, support
+
+Overview
+--------
+
+On this page, you can find a list of MongoDB and {+framework+} features that
+are not supported by the {+provider-long+}. Because the provider is in active
+development, some features listed on this page might be considered for future
+releases based on customer demand.
+
+MongoDB Features
+----------------
+
+The following sections discuss MongoDB features that are not supported by the {+provider-short+}.
+
+Time Series
+~~~~~~~~~~~
+
+The {+provider-short+} does not support time series data or time series
+collections. These are MongoDB-exclusive features that are not available in {+framework-core+}.
+
+Atlas Search
+~~~~~~~~~~~~
+
+The {+provider-short+} does not support MongoDB Atlas search. Atlas Search is a
+MongoDB-exclusive feature with no similar feature available in {+framework-core+}.
+
+Vector Search
+~~~~~~~~~~~~~
+
+The {+provider-short+} does not support MongoDB Atlas Vector search. Atlas Vector Search is a
+MongoDB-exclusive feature with no similar feature available in {+framework-core+}.
+
+{+framework-core+} Features
+------------------------------
+
+The following sections discuss {+framework-core+} features that are not
+supported by the {+provider-short+}.
+
+Migrations
+~~~~~~~~~~
+
+Migrations in {+framework-core+} are designed for relational databases. Because
+MongoDB is a document database, migrations are not supported by the
+{+provider-long+} at this time.
+
+Database-First Development
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB is designed to be flexible and does not require a database schema.
+Because of this schemaless design, database-first development is not supported
+in the {+provider-short+}. 
+
+Foreign Keys
+~~~~~~~~~~~~
+
+Because MongoDB is a document database, the {+provider-short+} does not support foreign keys.
+
+Alternate Keys
+~~~~~~~~~~~~~~
+
+Because MongoDB is a document database, the {+provider-short+} does not support alternate keys.
+
+Table Splitting
+~~~~~~~~~~~~~~~
+
+MongoDB is a document database and does not have normalized tables.
+Because of this, table splitting has limited use and is not supported by the
+{+provider-short+}.
```

Comment:
Q: in this context, what does limited use mean? Is table splitting supported in any capacity?

Reply:
In this context it was supposed to mean that it would only be somewhat useful if it were to be supported. I removed that part to make it more straightforward.